### PR TITLE
=str #16913 enable and fix broken StreamTcpDocSpec

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala [hunk 1]
```diff
@@ -54,29 +54,7 @@ class StreamTcpDocSpec extends AkkaSpec {
...
-      .map(text => println("Server: " + text))
-      .map(_ => readLine("> "))
-      .map {
-        case "q" =>
-          sys.shutdown(); ByteString("BYE")
-        case text => ByteString(s"$text")
-      }
-
-    connection.join(repl)
-    //#repl-client
```
Comment (patriknw) [inline akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala]: #repl-client is also defined below, so it was included twice

File: akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala [hunk 2]
```diff
@@ -150,7 +128,7 @@ class StreamTcpDocSpec extends AkkaSpec {
       .map(_ => readLine("> "))
       .transform(() ⇒ replParser)
 
-    connection.join(repl)
+    connection.join(repl).run()
```
Comment (rkuhn) [inline akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala]: d’oh

**Comment by rkuhn** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM, nice catch

**Comment by ktoso** [comment]
Good catch, LGTM

**Pull request merged**